numpy/core/code_generators/generate_umath.py

```diff
@@ -45,14 +45,19 @@ class TypeDescription:
     astype : dict or None, optional
         If astype['x'] is 'y', uses PyUFunc_x_x_As_y_y/PyUFunc_xx_x_As_yy_y
         instead of PyUFunc_x_x/PyUFunc_xx_x.
+    cfunc_alias : str or none, optional
+        replaces the suffix of C function name instead of using ufunc_name,
+        e.g. "FLOAT_{cfunc_alias}" instead of "FLOAT_{ufunc_name}" (see make_arrays)
```

Comment:
appended to inner loop C function name, e.g. ``FLOAT_{cfunc_alias}`` (see ``make_arrays``)